docker/build.gradle

```diff
@@ -42,11 +42,13 @@ task officialBuild {
         file(outputDir).mkdir()
 
         def rundeckapp = project(":rundeckapp")
+        def archiveFile = rundeckapp.war.archiveFile.get().toString().replace("-plain","")
+        def archiveFileName = rundeckapp.war.archiveFileName.get().replace("-plain","")
```

Comment:
I believe you can use `rundeckapp.bootWar.archiveFile` instead of getting the name of the plain war and removing the `-plain` part